Makefile

```diff
@@ -158,3 +158,18 @@ run-deeplearning:
 	docker run -it --rm $(DOCKER_USER)/compute-deeplearning$(ARCH):$(IMAGE_TAG) bash
 
 
+anaconda:
+	cd src/anaconda && docker build -t $(DOCKER_USER)/compute-anaconda:$(IMAGE_TAG) .
```

Comment:
You need to name the image with $(ARCH) in the name -- see all the other examples.
Basically this stuff needs to work on both x86 and arm64. Same with colab's image below.   If every line doesn't have $(ARCH) in it, then it is wrong.

Reply:
i'll fix it -- since everything else looks good and this is trivial to fix.